src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -186,6 +186,13 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+       public override void Reset()
+        {
+            this.PRTitle = string.Empty;
+            this.Description = string.Empty;
```

Comment:
You don't need the `this.` qualifiers here, that is only required in constructors when you need to distinguish between a constructor argument and a field or property that have the same name.